query-coordinator/src/main/scala/com/socrata/querycoordinator/fusion/CompoundTypeFuser.scala

```diff
@@ -141,6 +141,8 @@ class CompoundTypeFuser(fuseBase: Map[String, String]) extends SoQLRewrite with
           case None =>
             Some(expr)
         }
+      case fc@FunctionCall(fnName, Seq(ColumnOrAliasRef(name: ColumnName)))
+        if (Set(SoQLFunctions.PointToLatitude.name, SoQLFunctions.PointToLongitude.name).contains(fnName)) => Some(fc)
```

Comment:
Hrm, this bothers me on a few levels.  Mostly to do with making decisions on the function's `name` (since soql supports overloading) instead of its `identity`.  But I see that's happening in other places in this code already...